src/modules/0.0.5/aws_s3/entity/index.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  Column,
+  Entity,
+  PrimaryColumn,
+} from 'typeorm'
+
+import { cloudId, } from '../../../../services/cloud-id'
+
+@Entity()
+export class Bucket {
+  @PrimaryColumn({
+    nullable: false,
```

Comment:
I think primary columns are nullable and unique by definition so this is a no-op